### DISCUSSION TITLE
Is there really no way to see a video's URL and/or open the video in a browser?

### DISCUSSION THREAD

**Discussion by LinuxOnTheDesktop** [FreeTubeApp/FreeTube#3014] [Q&A] [answered]
Sometimes I want the URL of a video that I watch (so that I can refer to the URL later, or give the link to someone). So far as I can tell, the current version of FreeTube does not allow display of the URL, and does not even allow the URL's indirect display via the opening of a video in a browser.

**Comment by ChunkyProgrammer** [comment, marked as answer]
Hello, you should be able to copy the link or open in a browser. (There should be a share icon to click under the video)
![](https://user-images.githubusercontent.com/78101139/210286724-46e002d3-4454-4652-bcf3-c14df48301cd.png)

If you can't find the icon:
In Settings, under distraction free, ensure that 'Hide Sharing Actions' is toggled off

![](https://user-images.githubusercontent.com/78101139/210286787-8c7a77bf-fed2-4029-a127-669ad953325f.png)

Hope this helps 😃

**Comment by LinuxOnTheDesktop** [comment]
What a well-documented answer! Thank you! I found the requisite option within an instance of version 0.18 of the program, running on Windows. Oddly, when using version 0.17, on a Linux computer, 'Hide Sharing Actions' was missing from the 'Distraction Free Settings'. But when I upgraded that PC to 0.18 of the program, the option was there.

**Reply by ChunkyProgrammer** [reply]
I believe that setting was added in 0.17.1 🙂
